tests/test_basic.py

```diff
@@ -32,6 +32,24 @@ def test_total_basic(self):
                                    self.ecc.raw_pubkey)
         self.assertEqual(msg, msg2)
 
+    def test_inequal_slots(self):
+        msg1 = message.RandVal(rand_val=1)
+        msg2 = message.RandVal(rand_val=2)
+        self.assertNotEqual(msg1, msg2)
+
+    def test_inequal_header(self):
```

Comment:
@jiivan shouldn't there be a positive test for equality here? or are we counting `test_total_basic` as one?